Sources/ForageSDK/ForageSDK.swift

```diff
@@ -42,6 +42,13 @@ public class ForageSDK {
         // TODO: Maybe move this shared logger call!
         VGSCollectLogger.shared.disableAllLoggers()
         self.service = LiveForageService()
+        SentrySDK.start { options in
+            options.dsn = "https://8fcdd8dc94aa892ed8fd4cdb20db90ee@o921422.ingest.sentry.io/4505665631813632"
```

Comment:
Should this be added in secrets manager. We do have the same dsn for all environments.

Reply:
no secrets manager needed here since the DSN is safe to expose to the client-side